### PR TITLE
[WC-2026] Refactor pluggable commons 4

### PR DESCRIPTION
**Pull request by iobuhov**
### Pull request type

Refactoring (e.g. file rename, variable rename, etc.)

---

### Description
Same as before, removing PWC from deps.

### CODE REVIEW

File: packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { createElement, ReactElement, useEffect, useState, useCallback } from "react";
-import { Alert } from "@mendix/pluggable-widgets-commons/components/web";
+import { Alert } from "@mendix/widget-plugin-component-kit/Alert";
```
Comment (r0b1n) [inline packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx:2]: NIT: Would it be nicer to make this a default export?

**Review by r0b1n — DISMISSED** [review]

File: packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx (hunk 1 above)
Comment (iobuhov) [reply packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx:2]: default export for component?
There could be other components (like IconInternal)

Comment (iobuhov) [reply packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx:2]: In short, I would prefer to stay with named exports, as we have many tools in our stack that may treat default exports differently. I've already seen many issues related to default exports. If you need source, I can find, but for now let's just keep it as a named import.

**Review by gjulivan — APPROVED** [review]

File: packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx (hunk 1 above)
Comment (r0b1n) [reply packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx:2]: I was thinking that main component (same as filename) could be default while secondary can be named. But keeping as is is also fine.

**Review by r0b1n — APPROVED** [review]

**Pull request merged**